### DISCUSSION TITLE
use C51 toolchain in linux

### DISCUSSION THREAD

**Discussion by lgzcoollg** [xmake-io/xmake#5818] [Q&A]
由于我没有网络，请原谅我的是截图
**make.lua:**
<img width="923" alt="image" src="https://github.com/user-attachments/assets/801cbede-6e0a-44a0-b66e-ab1bdc39f001">

verb
<img width="1195" alt="image" src="https://github.com/user-attachments/assets/0bd397b2-5de8-4ed0-9984-14511621cc8e">

**Comment by waruqi** [comment]
目前 keil 系，我只对 win 做了支持，其他平台还不支持

**Reply by lgzcoollg** [reply]
好的，我现在是手工在wine下跑的C51.exe/BL51.exe，命令前加了wine，本质是也是在windows上跑 =)))

**Reply by waruqi** [reply]
不一样，这边内部还有很多对 windows 环境的处理，不单单只是这个 bin 本身。

**Reply by star-hengxing** [reply]
感觉可以交叉编译一个 xmake.exe ，扔进 wine 一起跑或者用远程编译功能？
